mobile-workspace/latest/using/index.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Using Mobile Workspace
+---
+
+## Searching
+
+* You can search globally from all of the main/top level screens which means you can search at any time.
+* The Mobile Workspace keeps a recent history of search terms so that previous searches can be quickly repeated.
+* Contextual search is available within folders. This search mode will only search within that folder but can be converted to a global search using the quick filter. You can also filter by files and folders within the contextual folder search.
+
+## Recent Files
+
+The Recent Files view shows all the files that have been created or modified within the last 30 days by the logged in user. The Search API is used to query for changes made by you and includes an extra column to display where the file is located in the content repository.
```

Comment:
I'd suggest using `**Recent Files**` for this and any other UI term in the rest of this page.

Reply:
ok, done.

Reply:
I'm writte nthe  **text descriptions** for UI items in bold, and reverted the changes to the headings.